modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/RocksDbTableStorage.java

```diff
@@ -401,11 +399,16 @@ int getTableId() {
         return tableDescriptor.getId();
     }
 
+    private void checkPartitionExists(int partitionId) {
+        if (mvPartitionStorages.get(partitionId) == null) {
```

Comment:
Is it different from `contains`? If the answer is `yes` then please add a comment

Reply:
What `contains` method are you talking about? Otherwise, the semantics is the same